dvc/logger.py

```diff
@@ -141,6 +141,26 @@ def _stack_trace(exc_info):
     )
 
 
+def disable_other_loggers():
+    allowed_loggers = [
+        "dvc",
+        "paramiko",
+        "flufl.lock",
```

Comment:
Why are we allowing these though? Paramiko is very noisy.

Reply:
I took the list from declared [loggers](https://github.com/iterative/dvc/blob/a9bc65ee1f0446de766db59ad1b149de064c5360/dvc/logger.py#L177)
I got that pointing specific log level for `paramiko` and `flufl.lock` can be done for hiding a lot of messages from them, but wasn't sure
Anyway the output of them will be handled by specific local loggers as was previous

Reply:
But if we disable them here, then we won't need to tweak them in logger.py, right? If so, let's just disable them here, as it was the original intention.